docs/docs/recipes.md

```diff
@@ -122,13 +122,53 @@ To wrap pages with layouts, use normal React components.
 
 ## Deploying
 
-Showtime.
+Showtime. Once you are happy with your site, you are ready to go live with it!
+
```

Comment:
```suggestion
### Preparing for deployment
```
I think we're missing a heading for the recipe here!

Reply:
Yeah, wasn't sure on the format for that, thanks for adding it in!